Build failure: `hypr-dynamic-cursors`

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

1. Update flake
2. Rebuild

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/307560878

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/fpwvrh9z69nal4l7xz41iqjbwvx2qmrm-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: SHELL=/nix/store/q7sqwn7i6w2b67adw0bmix29pxg85x3w-bash-5.3p3/bin/bash
g++ -Wall --no-gnu-unique -fPIC -std=c++26 -g -I/nix/store/h9qsz8b1mbih1m2804a00y9phg2gcn7d-hyprland-0.51.0-dev/include/hyprland/src -I/nix/store/h9qsz8b1mbih1m2804a00y9phg2gcn7d-hyprland-0.51.0-dev/include -I/nix/store/h9qsz8b1mbih1m2804a00y9phg2gcn7d-hyprland-0.51.0-dev/include/hyprland/protocols -I/nix/store/h9qsz8b1mbih1m2804a00y9phg2gcn7d-hyprland-0.51.0-dev/include/hyprland -I/nix/store/ssshfj3zgwnx3j77spypa7c5v5ixc5sc-pixman-0.46.2/include/pixman-1 -I/nix/store/myrhngjdaz6fxk4izl3phi0l0p7zir61-libdrm-2.4.125-dev/include -I/nix/store/myrhngjdaz6fxk4izl3phi0l0p7zir61-libdrm-2.4.125-dev/include/libdrm -c src/cursor.cpp -o out/cursor.o
src/cursor.cpp: In member function 'SP<Aquamarine::IBuffer> CDynamicCursors::renderHardware(CPointerManager*, SP<CPointerManager::SMonitorPointerState>, SP<CTexture>)':
src/cursor.cpp:226:116: error: 'class CCompositor' has no member named 'm_drmFD'; did you mean 'm_drm'?
  226 |         options.multigpu = state->monitor->m_output->getBackend()->preferredAllocator()->drmFD() != g_pCompositor->m_drmFD;
      |                                                                                                                    ^~~~~~~
      |                                                                                                                    m_drm
make: *** [Makefile:21: out/cursor.o] Error 1
```

### Additional context

PR #442786 might fix it?

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.5-zen1, NixOS, 25.11 (Xantusia), 25.11.20250908.b599843`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Determinate Nix 3.11.1) 2.31.1`
 - nixpkgs: `/nix/store/4bdm5fcwix168v4l6rr9p7a9iafgp6zs-source`

### Notify maintainers

@donovanglover

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/window-managers/hyprwm/hyprland-plugins/hypr-dynamic-cursors.nix
+++ b/pkgs/applications/window-managers/hyprwm/hyprland-plugins/hypr-dynamic-cursors.nix
@@ -8,13 +8,13 @@
 
 mkHyprlandPlugin hyprland {
   pluginName = "hypr-dynamic-cursors";
-  version = "0-unstable-2025-07-19";
+  version = "0-unstable-2025-09-01";
 
   src = fetchFromGitHub {
     owner = "VirtCode";
     repo = "hypr-dynamic-cursors";
-    rev = "d6eb0b798c9b07f7f866647c8eb1d75a930501be";
-    hash = "sha256-Yd5oSg1gS/mwobd5YFrLC3I4bar/cSGNGuIvxF3UeHE=";
+    rev = "0e11ed12dbd4d0c62b362dda2559c2e374814d56";
+    hash = "sha256-2JQ/KxMEVDjauaFU62T0udjKXKSiCyKO5frjVf4b4A8=";
   };
 
   dontUseCmakeConfigure = true;

